networkx/drawing/layout.py

```diff
@@ -1111,6 +1112,218 @@ def multipartite_layout(G, subset_key="subset", align="vertical", scale=1, cente
     return pos
 
 
+def forceatlas2_layout(
+    G,
+    pos=None,
+    n_iter=100,
+    jitter_tolerance=1.0,
+    scaling_ratio=2.0,
+    gravity=1.0,
+    distributed_action=False,
+    strong_gravity=False,
+    adjust_sizes=False,
+    dissuade_hubs=False,
+    edge_weight_influence=False,
+    linlog=False,
+    dim=2,
+):
+    """Forceatlas2 layout for networkx
+
+    See [1]_ for more info on the parameters
+
+    .. [1] https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0098679&type=printable
```

Comment:
move the reference to a References section at the end of the doc_string. And can we at least put together a few words about this article to describe it rather than just "See [1]_ for`. Say who did it or even that this function implements the algorithm in that paper.

Consider bulking up the parameter descriptions. Nobody likes to be referred to a publication when a two sentence description is all they are looking for. And this sentence makes it appear that we're just giving up on describing parameters.